doc/generate_cet.py

```diff
@@ -3,7 +3,144 @@
 import pathlib
 
 
+def provide_ignored():
+    """Returns lists of ignored attributes and fields."""
+    ignored_attributes = ["EFDB_Name", "Description"]
+    ignored_fields = ["Description", "Alias", "Device", "Property", "Action", "Value", "Subsystem", "Version", "Author", "Explanation"]
+    return ignored_attributes, ignored_fields
+
+def add_generics(cf, subsystem, set_name, has_generics, has_specific):
+    """Adds generic topics to the rst file in the appropriate section.
+
+    Parameters
+    ----------
+    cf : `IO`
+        A file object for writting to each CSC rst file.
+    subsystem : `str`
+        The name of the CSC.
+    set_name : `str`
+        The name of the topic set.
+    has_generics : `bool` or `List` of `str`
+        It either has generic topics(yes) or not(no) or a list of supported topics.
```

Comment:
Please consider something along the lines of "True if it has all generic topics, False if it has no generic topics, a list of generic topic names if it has some generic topics."